.github/workflows/platformDeploy.yml

```
@@ -253,7 +253,7 @@ jobs:
 
       - name: Upload iOS build to GitHub Release
         if:  ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
-        run: gh release upload ${{ github.event.release.tag_name }} /Users/runner/work/App/App/New Expensify.ipa
+        run: gh release upload ${{ github.event.release.tag_name }} /Users/runner/work/App/App/New\ Expensify.ipa
```

this works, but a perhaps simpler alternative could have been:

```suggestion
        run: gh release upload ${{ github.event.release.tag_name }} '/Users/runner/work/App/App/New Expensify.ipa'
```